ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main__tempo-product-ocp-4.15-lp-interop.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
```

Comment:
Does the tag need to be updated to reflect OpenShift 4.15?

Reply:
I don't believe that it needs to be, this image is just used to build other images from what I know, I wanted to leave it as is since we know it is valid. I'm not sure about a possible -4.15 of this image and I didn't want to break the scenario due to that.

If anyone who knows more about this scenario then me can provide details that would be good too.

Reply:
going to update and test these